### PR TITLE
Added brand for Kilwins

### PR DESCRIPTION
**Pull request by zbc**

### CODE REVIEW

File: brands/shop/confectionery.json [hunk 1]
```diff
@@ -39,6 +39,19 @@
       "shop": "confectionery"
     }
   },
+  "shop/confectionery|Kilwins": {
+    "countryCodes": ["us"],
+    "matchTags": ["amenity/ice_cream"],
+    "tags": {
+      "alt_name": "Kilwin's",
```
Comment (bhousel) [inline brands/shop/confectionery.json]: We currently can't support having the alternate names very similar to the main names (different only in spaces, case, punctuation), so I am going to remove the `alt_name`, but other than that the PR looks ok 👍 

There haven't been any other brands where this been an issue, but I could probably change the matching code to support it if it matters a lot to people.  We don't use `alt_name` much so it hasn't been a problem yet.

File: brands/shop/confectionery.json [hunk 2]
```diff
@@ -39,6 +39,19 @@
...
+      "brand": "Kilwins",
+      "cuisine": "chocolate;ice_cream;popcorn",
+      "name": "Kilwins",
+      "shop": "confectionery",
+      "wikidata": "Q38250832",
```
Comment (bhousel) [inline brands/shop/confectionery.json]: oh!  also these need to be `brand:wikidata` and `brand:wikipedia`.. I'll fix that too.

File: brands/shop/confectionery.json (hunk 1 above)
Comment (zbc) [reply brands/shop/confectionery.json]: Got it.

File: brands/shop/confectionery.json (hunk 2 above)
Comment (zbc) [reply brands/shop/confectionery.json]: Sorry about that. Thanks.

**Comment by bhousel** [comment]
Oh one other thing - if you add a text like "closes #2728" to the commit message, it will automatically close the issue once the commit is merged..  Pretty convenient!

https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by zbc** [comment]
> Oh one other thing - if you add a text like "closes #2728" to the commit message, it will automatically close the issue once the commit is merged.. Pretty convenient!
> 
> https://help.github.com/en/articles/closing-issues-using-keywords

Wow!! Good to know. Thanks!! Will do next time.

**Pull request merged**